packages/grid/src/InternalGrid.tsx

```diff
@@ -86,18 +87,17 @@ const InternalGrid: React.FC<IInternalGrid> = ({
     const newFullWidth = getFullWidth(gridHOCMappedColumns);
 
     fullWidthRef.current = newFullWidth;
```

Comment:
This part of code repeats in this useEffect three times: `fullWidthRef.current = newFullWidth;`
I think only this one is necessary.